pinot-common/src/main/java/org/apache/pinot/common/metrics/ControllerGauge.java

```diff
@@ -132,7 +132,16 @@ public enum ControllerGauge implements AbstractMetrics.Gauge {
   MAX_RECORDS_LAG("maxRecordsLag", false),
 
   // Consumption availability lag in ms at a partition level
-  MAX_RECORD_AVAILABILITY_LAG_MS("maxRecordAvailabilityLagMs", false);
+  MAX_RECORD_AVAILABILITY_LAG_MS("maxRecordAvailabilityLagMs", false),
+
+  // Number of table schema got fixed
+  FIXED_SCHEMA_TABLE_COUNT("FixedSchemaTableCount", true),
```

Comment:
Do we have these configs covered in the prometheus yaml?

Reply:
those should be covered by the match all rule